docs/connector.md

```diff
@@ -0,0 +1,51 @@
+# Connector
+
+The [`backend`](./architecture.md#backend) includes an [Interledger](./glossary.md#interledger-protocol) connector for sending and receiving [STREAM](./glossary.md#stream) packets.
+
+The amounts of these packets are used to update account balances in [TigerBeetle](./glossary.md#tigerbeetle). (See [Accounts and Transfers](./accounts-and-transfers.md))
+
+Amounts are adjusted based on the destination/outgoing account's asset and Rafiki's configured exchange [rates service](./integration.md#rates-prices).
+
+## Packet Origination
+
+The connector receives incoming ILP packets via:
+
+- HTTP
+- direct calls from within the `backend`
```

Comment:
That is when the outgoing payment sends to an incoming payment on the same Rafiki, right? Can we maybe add that in a half-sentence?

Reply:
I think outgoing payments send packets to the connector for both local and remote incoming payment stream connections

Reply:
Both HTTP and local backend originating packets can be addressed to either local destination or outgoing peer.
So "direct calls from within the `backend`" includes but isn't limited to intra-Rafiki payments.